workspaces/playlist/plugins/playlist/src/components/EntityPlaylistDialog/EntityPlaylistDialog.test.tsx

```diff
@@ -95,6 +109,10 @@ describe('EntityPlaylistDialog', () => {
             [alertApiRef, alertApi],
             [permissionApiRef, permissionApi],
             [playlistApiRef, playlistApi],
+            [
+              entityPresentationApiRef,
```

Comment:
With the change to use `EntityDisplayName` instead, I don't think you need to include these mocks any more to get the tests passing - you can also remove the `@backstage/plugin-catalog` dependency as well if you revert these changes